it/mail/chrome/messenger/preferences/display.dtd

```diff
@@ -50,7 +50,7 @@
 <!ENTITY pb.conversations.sortoldest.label "Prima i messaggi meno recenti">
 <!ENTITY pb.conversations.sortnewest.label "Prima i messaggi più recenti">
 
-<!ENTITY pb.conversations.read.label "Quando si visualizza una conversazione, contrassegnare i suoi messaggi come già letti">
+<!ENTITY pb.conversations.read.label "Quando visualizzi una conversazione, contrassegnare i suoi messaggi come già letti">
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

+<!ENTITY pb.conversations.read.label "Quando visualizzi una conversazione, contrassegna i suoi messaggi come già letti">